feature/experimental/isis/ate_tests/isis_change_lsp_lifetime_test/README.md

```diff
@@ -0,0 +1,44 @@
+## RT-2.10: IS-IS change LSP lifetime
+
+## Summary
+
+* Changing the lsp lifetime and verifying isis lsp parametrs
+
+## Procedure
+
+* TestChangeLspLifetime
+
+    * Configure IS-IS for ATE port-1 and DUT port-1.
+    * Modify the default life time of the LSP PDU.
```

Comment:
the text alternates between lifetime as one word or two. Standardize to one word

Reply:
addressed in latest commit